sample_workloads/nccltest/docker/scripts/container_entry.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+# Container entry script. Runs NCCL-level benchmarks and report results.
+
+set -u
+
+# Job parameters.
+: "${JOB_TIMESTAMP:?Must set JOB_TIMESTAMP}"
+: "${JOB_NAME:?Must set JOB_NAME}"
+: "${MASTER_ADDR:?Must set MASTER_ADDR}"
+: "${NNODES:?Must set NNODES}"
+: "${NODE_RANK:?Must set NODE_RANK}"
+
+# GCS bucket to be used.
+: "${GCS_BUCKET:?Must set GCS_BUCKET}"
+
+# Benchmark parameters.
+: "${BENCHMARKS_CSV:?Must set BENCHMARKS_CSV}"
+: "${MASKS_CSV:?Must set MASKS_CSV}"
+: "${MSG_SIZE_BEGIN:?Must set MSG_SIZE_BEGIN}"
+: "${MSG_SIZE_END:?Must set MSG_SIZE_END}"
+: "${GPUS_PER_NODE:?Must set GPUS_PER_NODE}"
+: "${N_COMMS:?Must set N_COMMS}"
+: "${WARMUP_ITERS:?Must set WARMUP_ITERS}"
+: "${RUN_ITERS:?Must set RUN_ITERS}"
+: "${N_RUNS:?Must set N_RUNS}"
+
+# Unreserved cores for taskset call. This is a CSV of ranges for cores unused
+# by TCPX.
+: "${UNRESERVED_CORES:?Must set UNRESERVED_CORES}"
+
+# Telemetry.
+: "${GPU_TELEMETRY:?Must set GPU_TELEMETRY}"
```

Comment:
Remove this.

Reply:
Any specific reason we want to remove the Nsight support? 

We have nsight traces exposed as optional in our other sample workflows.